modules/cnf-fast-event-notifications-api-refererence.adoc

```diff
@@ -174,11 +150,12 @@ OK
 [discrete]
 ==== Description
 
-Returns an array of `os-clock-sync-state`, `ptp-clock-class-change`, and `lock-state` details for the cluster node. The system generates notifications when the relevant equipment state changes.
+Returns an array of `os-clock-sync-state`, `ptp-clock-class-change`, `lock-state`, and `gnss-sync-status` details for the cluster node. The system generates notifications when the relevant equipment state changes.
 
 * `os-clock-sync-state` notifications describe the host operating system clock synchronization state. Can be in `LOCKED` or `FREERUN` state.
 * `ptp-clock-class-change` notifications describe the current state of the PTP clock class.
 * `lock-state` notifications describe the current status of the PTP equipment lock state. Can be in `LOCKED`, `HOLDOVER` or `FREERUN` state.
+* `gnss-sync-status` notifications describe the host operating system clock synchronization state with regard to the external GNSS clock signal. Can be in `LOCKED` or `FREERUN` state.
```

Comment:
gnss-sync-state describes the  gps syn state only , either GPS is locked , in sync or not

Reply:
Individual subscriptions to /sync/sync-status/sync-state and /sync/gnss-status/gnss-sync-status would deliver notifications for both the overall synchronization health.
event.sync.gnss-status.gnss-state- change Notification used to inform about gnss synchronization state change